ocs_sample_library_preview/SDS/SdsResultPage.py

```diff
@@ -0,0 +1,58 @@
+import json
+from typing import Any
+
+
+class SdsResultPage(object):
+    """
+    definition of SdsResultPage
+    """
+
+    def __init__(self, results: list[Any] = None, continuation_token: str = ''):
+        self.Results = results
+        self.ContinuationToken = continuation_token
+
+    @property
+    def Results(self) -> list[Any]:
+        return self.__results
+
+    @Results.setter
+    def Results(self, value: list[Any]):
+        self.__results = value
+
+    @property
+    def ContinuationToken(self) -> str:
+        return self.__continuation_token
+
+    @ContinuationToken.setter
+    def ContinuationToken(self, value: str):
+        self.__continuation_token = value
+
+    def toJson(self):
+        return json.dumps(self.toDictionary())
+
+    def toDictionary(self):
+        result = {'Results': self.Results,
+                  'ContinuationToken': self.ContinuationToken}
+
+        return result
+
+    @staticmethod
+    def fromJson(content: dict[str, str]):
+        result = SdsResultPage()
+
+        if not content:
+            return result
+
+        if 'Results' in content:
+            results = content['Results']
+            result.Results = results
+
+        if 'ContinuationToken' in content:
+            result.ContinuationToken = content['ContinuationToken']
+        else:
```

Comment:
Else block is not necessary here, it will default to `None`

Reply:
I think it is necessary from when I tested it. In the last "page" no continuation token is returned in the response so it would not find 'ContinuationToken' in the json response. Therefore, the ContinuationToken will remain what it was before.

Reply:
But the `result` object was declared on line 41, and `ContinuationToken` is initialized to `None` there. There's no way that it would have a previous value, since you're not passing in a previous `SdsResultPage` to this function and it's a `@staticmethod`.

Reply:
I went ahead and changed the default initialization parameters and removed this line.